build.sh

```diff
@@ -856,7 +856,7 @@ if [ "$WITHOUT_TEST" = true ] ; then
 fi
 
 if [ "$DEBUG" = true ] ; then
-    echo "rpmbuild -bs ${OPTIONS[@]} $SPEC_FILE"
+    echo "rpmbuild -bs " + "${OPTIONS[@]}" + "$SPEC_FILE"
```

Comment:
Actually, I have always used the syntax above in shell code. For me is more readable but the new version is OK

Reply:
Do we need the `+` sign? Might be cleaner without it (but keep the new quotes).

Reply:
Ah yeah good point, I'll push that optimisation directly, thanks!